src/Components/Rockets.jsx

```diff
@@ -1,7 +1,89 @@
-import React from 'react';
+// IMPORTS
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import {
+  Button, Container, Row, Col,
+} from 'react-bootstrap';
+import { getRockets, reserveRocket, cancelRocket } from '../Redux/rocketsSlice';
+import '../styling/rockets.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+
+  // /GET ROCKETS FROM LOCAL STORAGE
+  const storedRockets = JSON.parse(localStorage.getItem('rockets')) || [];
+
+  // MERGE ROCKETS DATA WITH LOCAL STORAGE
+  const mergedRockets = rockets.map((rocket) => {
+    const storedRocket = storedRockets.find((r) => r.id === rocket.id);
+    if (storedRocket) {
+      return { ...rocket, reserved: storedRocket.reserved };
+    }
+    return rocket;
+  });
+
+  // Fetch rockets data on page load
+  useEffect(() => {
+    dispatch(getRockets());
+  }, [dispatch]);
+
+  // Handle reserve/cancel rocket
+  const handleReserveRocket = (id, reserved) => {
+    // Update the local storage and dispatch the action
+    if (reserved) {
+      localStorage.setItem(
+        'rockets',
+        JSON.stringify(mergedRockets.map((rocket) => (
+          rocket.id === id ? { ...rocket, reserved: false } : rocket))),
+      );
+      dispatch(cancelRocket(id));
+    } else {
+      localStorage.setItem(
+        'rockets',
+        JSON.stringify(mergedRockets.map((rocket) => (
+          rocket.id === id ? { ...rocket, reserved: true } : rocket))),
+      );
+      dispatch(reserveRocket(id));
+    }
+  };
+
+  // Show loading, error, or rockets data
+  const renderRockets = () => {
+    if (mergedRockets.length === 0) {
+      return <p>Loading rockets...</p>;
+    }
+
+    return mergedRockets.map((rocket) => (
+      <Container className="container-rocket" key={rocket.id}>
+        <Row>
+          <div className="img-rocket">
+            <img src={rocket.image} alt={rocket.image} />
+          </div>
+          <Col>
+            <h2>{rocket.name}</h2>
+            <Row className="reserved">
+              <div className="res">
+                {rocket.reserved ? (
+                  <h5>Reserved</h5>
+                ) : null}
+              </div>
+              <p>{rocket.description}</p>
```

Comment:
I would kindly suggest the Reserved label be placed in front of the paragraph when the reserve button is clicked. This will make the layout the same as the template provided.

Your current layout:
<img width="1013" alt="image" src="https://github.com/George7h/space-travelers/assets/101390796/4f65e84a-5629-498a-bc1c-7472e31caebf">

Their layout: 
![image](https://github.com/George7h/space-travelers/assets/101390796/d86c4899-33ae-4c34-a965-f1284e3f62e5)